sktime/forecasting/croston.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._sktime import _SktimeForecaster
+from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
+
+
+DEFAULT_SMOOTHING = 0.1
```

Comment:
I suggest to make this private `_DEFAULT_SMOOTHING` or if it's only used here just use the value directly without defining a variable for it